ai21/clients/common/assistant/thread.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import List
+
+from ai21.models.responses.thread_response import ThreadResponse, CreateMessagePayload
+
+
+class Thread(ABC):
```

Comment:
`Thread` or `Threads`?

Reply:
sounds good. I just went with the same naming convention of the previous PR...